support/agent/miner_agent.go

```diff
@@ -372,6 +417,11 @@ func (ma *MinerAgent) submitPoStForDeadline(v SimState, dlIdx uint64) ([]message
 		}
 	}
 
+	// schedule post-deadline state synchronization and next PoSt
+	if err := ma.scheduleSyncAndNextProof(v, dlIdx); err != nil {
```

Comment:
Move this to earlier in the call so that it still gets called when PoSt contains no sectors.